metadata.rb

```diff
@@ -4,6 +4,8 @@
 license          'GNU Public License 3.0'
 description      'Installs/Configures newrelic monitoring'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+issues_url       'https://github.com/chr4-cookbooks/newrelic-ng/issues'
+source_url       'https://github.com/chr4-cookbooks/newrelic-ng'
```

Comment:
This will break older chef-clients. Please use 

```ruby
issues_url 'https://github.com/chr4-cookbooks/newrelic-ng/issues' if respond_to?(:issues_url)
source_url 'https://github.com/chr4-cookbooks/newrelic-ng'  if respond_to?(:source_url)
```